src/components/Laatija/yritykset.svelte

```diff
@@ -57,14 +57,23 @@
     overlay = value;
   };
 
-  const parseYritys = name =>
-    R.isEmpty(R.trim(name))
-      ? Either.Right(Maybe.None())
-      : Yritys.findYritysByYtunnus(allYritykset, R.slice(0, 9, name))
```

Comment:
It seems that all references to Yritys got removed. Could remove the import as well.